enhancements/machine-config/install-time-support-for-image-mode-on-openshift.md

```diff
@@ -0,0 +1,343 @@
+---
+title: install-time-support-for-image-mode-on-openshift
+authors:
+  - "@umohnani8"
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - "@mrunalp"
+  - "@jlebon"
+  - "@cheesesashimi"
+  - "@yuqi-zhang"
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - "@mrunalp"
+  - "@jlebon"
+  - "@cheesesashimi"
+  - "@yuqi-zhang"
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - None
+creation-date: 2025-07-25
+last-updated: 2025-09-29
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/MCO-1347
+see-also:
+  - "https://github.com/openshift/enhancements/pull/1515/files?short_path=9f0c5f1#diff-9f0c5f1adabad0dfbdb3c9a5b66e53d4fc6619274d7a4c260508d148de17f5c1"
+replaces:
+  - ""
+superseded-by:
+  - ""
+---
+
+
+# Install Time Support for Image Mode on OpenShift
+
+## Summary
+
+This enhancement proposes adding install time support for Image Mode on Openshift, enabling cluster admins to customize the operating system layer during initial cluster installation.
+This feature allows users to build and deploy custom OS images containing additional software, drivers, and configurations from the very beginning of the cluster lifecycle, eliminating the need for post-installation customization and ensuring all nodes start with the desired configuration.
+
+## Motivation
+
+Image Mode on OpenShift became GA in OCP 4.18.21+ and has been well received as a post-install OS customization option. 
+
+We want to bring that support to install time so cluster admins can customize and configure their OS from the beginning instead of having to do it post-install.
+
+OpenShift's current approach to OS management provides excellent consistency and supportability through RHCOS (Red Hat CoreOS), but it requires cluster admins to cede certain configurability aspects to the platform. As workloads become more specialized there is an increasing need for OS-level customization that can be applied from Day 0. An obvious example today is how AI workloads require specific hardware drivers and configurations on nodes.
```

Comment:
nit: I think this is the only place we refer to "Day 0". Possibly better to be consistent and just call it install time support everywhere